roles/ironic-data/meta/main.yml

```diff
@@ -1,9 +1,3 @@
 ---
 dependencies:
   - role: ironic-common
-  - role: nova-data
```

Comment:
Why was this section removed? It looks like there is no longer anywhere that sets nova.compute_driver